packages/cloud_functions/cloud_functions_web/lib/cloud_functions_web.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:cloud_functions_platform_interface/cloud_functions_platform_interface.dart';
+import 'package:firebase/firebase.dart' as firebase;
+import 'package:flutter/foundation.dart';
```

Comment:
unused import?

Reply:
foundation.dart is the source of the @required annotation in the callCloudFunction method signature